tutorials/Tutorial_For_QA_Registration.ipynb

```diff
@@ -0,0 +1,377 @@
+    "                atl = ndimage.rotate(atlas[pos, :, :], 90)\n",
+    "            elif i == 1:\n",
+    "                image = ndimage.rotate(b0[:, pos, :], 90)\n",
+    "                atl = ndimage.rotate(atlas[:, pos, :], 90)\n",
+    "            else:\n",
+    "                image = b0[:, :, pos]\n",
+    "                atl = atlas[:, :, pos]\n",
+    "                \n",
+    "            if idx % 3 == 1:\n",
+    "                ax.set_ylabel(labs[i])\n",
+    "                ax.yaxis.set_ticks([0, image.shape[0] / 2, image.shape[0] - 1])\n",
+    "                ax.xaxis.set_ticks([0, image.shape[1] / 2, image.shape[1] - 1])\n",
+    "            if edge:\n",
+    "                image = edge_map(image).data\n",
+    "                image[image > 0] = max_val\n",
+    "                image[image == 0] = min_val\n",
+    "            #Set the axis invisible\n",
+    "            plt.xticks([])\n",
+    "            plt.yticks([])\n",
+    "            \n",
+    "            #Set the frame invisible\n",
+    "            ax.spines['top'].set_visible(False)\n",
+    "            ax.spines['right'].set_visible(False)\n",
+    "            ax.spines['bottom'].set_visible(False)\n",
+    "            ax.spines['left'].set_visible(False)\n",
+    "            \n",
+    "            ax.imshow(atl, interpolation=\"none\", cmap=cmaps[0], alpha=0.9)\n",
+    "            ax.imshow(\n",
+    "                opaque_colorscale(\n",
+    "                    cmaps[1], image, alpha=0.9, vmin=min_val, vmax=max_val\n",
+    "                )\n",
+    "            )\n",
+    "            \n",
+    "            #set the legend\n",
+    "            if idx == 3:\n",
+    "                plt.plot(0,0,\"-\",c=\"pink\",label='registered')\n",
+    "                plt.plot(0,0,\"-\",c=\"green\",label='reference')\n",
+    "                plt.legend(loc='best',fontsize=12,bbox_to_anchor=(1.5,1.5))\n",
+    "            \n",
+    "    #Set title for the whole picture\n",
+    "    a,b,c = brain_volume\n",
+    "    title = 'QA For Registration. Brain Volume:'+ str(a) +'*'+ str(b) + '*' + str(c)+'\\n'\n",
+    "    foverlay.suptitle(title,fontsize=24)\n",
+    "    foverlay.set_size_inches(12.5, 10.5, forward=True)\n",
+    "    foverlay.tight_layout()\n",
+    "    return foverlay"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Input and Graphs\n",
+    "Change location of input_1, input_2 and output_dir after running ndmg on desired dataset to your output folder"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Anatomical  Registration \n",
+    "### Parameters:\n",
+    "input_1: str\n",
+    "    \n",
+    "    the registered output of the intermediate steps of the registration.\n",
+    "\n",
+    "input_2: str\n",
+    "    \n",
+    "    the reference image used in Registration step.\n",
+    "\n",
```

Comment:
Change `input_1` to `mri` and `input_2` to `atlas` to make this consistent with `reg_mri_pngs`

Reply:
see https://github.com/neurodata/ndmg/pull/372/commits/7b32b3d114ee8d7a65060fb0dc1fefa8393cb052